pkgs/development/libraries/libspatialaudio/default.nix

```diff
@@ -0,0 +1,21 @@
+{ lib, stdenv, fetchFromGitHub, cmake, libmysofa, zlib }:
```

Comment:
Sorry about this, but this looks really strange. Like a giraffe without neck or something.

```suggestion
{ lib, stdenv, fetchFromGitHub, cmake, libmysofa, zlib }:

```

Reply:
Hah, someone needs to add this to nixfmt :P